### DISCUSSION TITLE
Autocomplete issue

### DISCUSSION THREAD

**Discussion by RealityMasque** [Megabit/Blazorise#5037] [General]
Hi all,

Is anyone else experiencing the FreeTypingNotFoundTemplate not rendering when Autocomplete.FreeTyping = true & the entered text is not in the data?  Here's some sample code to show what I tried:

```
<h3>AutocompleteTest</h3>

<Autocomplete TItem="string"
              TValue="string"
              Data="@colors"
              Placeholder="Search..."
              Filter="AutocompleteFilter.StartsWith"
              FreeTyping=true
              TextField="@(( item ) => item)"
              ValueField="@(( item ) => item)">
    <FreeTypingNotFoundTemplate>
        freetext not found
    </FreeTypingNotFoundTemplate>
</Autocomplete>

@code {
    private List<string> colors = new List<string>
    {
        "Red",
        "Orange",
        "Yellow",
        "Bright Green",
        "Dark Green",
        "Sky Blue",
        "Blue",
        "Purple",
        "Hot Pink",
        "Black",
        "White",
        "Gray"
    };
}
```

Thanks,

Orion

**Comment by RealityMasque** [comment]
I also checked issues in the repo & didn't find anything related to FreeTypingNotFoundTemplate not showing.

**Reply by David-Moreira** [reply]
creating an issue so we can troubleshoot if there's an issue.

**Reply by RealityMasque** [reply]
thanks!

**Reply by RealityMasque** [reply]
@David-Moreira - do you know if the FreeTypingNotFoundTemplate is intended to allow other components in it?  the NotFoundContent shows up in it's own drop down (like when you find something), but it's content is not interactable.  we need to allow users to click a button to request a new item when what they are looking for is not found.

**Reply by David-Moreira** [reply]
The behaviour is the same as the `NotFoundContent`, razor/html structure wise. So any limitations would be the same.
I believe we had a similar complaint before, I'll get back to you on this one.
